source/frontend/src/components/maps/hooks/useMapSearch.tsx

```diff
@@ -38,7 +38,7 @@ export const useMapSearch = () => {
           lat: +filter.latitude,
           lng: +filter.longitude,
         });
-        const task2 = pimsService.findOneWhereContains(
```

Comment:
Really this should be against the boundary as that is much more likely to return a result. In the future when we have a method of populating the boundary as well as the location this will be even more likely to return results (currently this will only return etl'd results).